src/main/java/tc/oc/pgm/rotation/MapPoll.java

```diff
@@ -225,4 +228,32 @@ private void updateScores() {
     if (voters == 0) return;
     votes.forEach((m, v) -> mapScores.put(m, Math.max(v.size() / voters, Double.MIN_VALUE)));
   }
+
+  private void announceMapWinner(Match match, MapInfo winner) {
```

Comment:
I would just merge this method & the `sendMessage` method, wich is already done per-player, just rename `sendMessage` to `announceMapWinner` with the player & picked map as params, and integrate this logic there, it's only called when the map has actually ended.

You could also make it so the picked map name is underlined or bolded in the chat msg

Reply:
Awesome,
I was initially unsure whether it would be more appropriate to make this separate, however I will merge this into the sendMessage method. Thanks!

I’ll also make the formatting changes suggested 👍